smartcontractkit/external-adapters-js
# Feature/DF-19037 standardize lwba endpoints via ea framework

## Closes [DF-19037](https://smartcontract-it.atlassian.net/browse/DF-19037)

## Description
Updated all LWBA EAs (NCFX, Coinmetrics, CF benchmarks, Tiingo) to framework v0.33.0 in order to use the LwbaEndpoint and related types

See related framework PR: https://github.com/smartcontractkit/ea-framework-js/pull/233

## Changes

- Used LwbaEndpoint and related types where possible for solo LWBA endpoints
- For PriceEndpoints that have LWBA as an alias, used applicable types
- Removed unused fields for these (according to RDD Arbitrum mainnet)
- As a result, other EAs had to be upgraded (Kaiko, Coingecko, Coinpaprika, CryptoCompare, Coinmarketcap) due to non-deployable token-allocation-adapter
- Related updates to tests

## Steps to Test

1. Run unit/integration tests for changed EAs
2. Sanity test all EAs affected by this PR, particularly LWBA endpoints and crypto endpoints where applicable
3. Verify removed fields are indeed unused everywhere

## Quality Assurance



[DF-19037]: https://smartcontract-it.atlassian.net/browse/DF-19037?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
